config/routes.rb

```diff
@@ -11,6 +11,26 @@
         mount BrowseEverything::Engine => '/browse'
 
   mount Riiif::Engine => 'iiif/2', as: :riiif if Hyrax.config.iiif_image_server?
+
+  # Hide top-level routes to the dashboard when running the public access site.
```

Comment:
Should these redirects be the first routes defined so they are matched first? I'd worry that Bulkrax, BrowseEverything, etc might expose a route.

Reply:
Yes to that. Routes moved to very top and still function correctly. Notice I didn't use the suggested helper in the route logic - I'm unsure of the class loading order and whether routes might be drawn before the helper can be resolved.